frontend/src/components/common/dokumente/Dokumente.vue

```diff
@@ -99,11 +108,15 @@ export default class Dokumente extends Mixins(
     }
   }
 
-  private onFilesSelected(event: Event): void {
+  async onFilesSelected(event: Event): Promise<void> {
```

Comment:
Einfügen einer Methodendokumentation die beschreibt was in der Methode passiert.

z.B. das hier auch der Progress-Circle aktiviert bzw. deaktiviert wird.

Reply:
erledigt

Reply:
Mit Methodendokumentation  habe ich eher die Doku über der Methode gemeint.

Hier sollte noch ergänzt werden, dass sich die Methode auch um den Progress-Circle kümmert.

Reply:
Methodendokumentation nachgebessert